src/adapter_interface.js

```diff
@@ -0,0 +1,7 @@
+const assertParam = require('./utils/assert-param')
+
+module.exports.create = config => {
+    assertParam(config, 'provider')
```

Comment:
Could you consider using 2-space indentation? If you strongly prefer 4-space indentation I think it's just fine but otherwise I think it could be advantageous to have the same indentation style as other repos such as ain-blockchain and ain-js?

Reply:
I will follow the rules  :) Thanks for reminding me of this. If there's coding standards in any written form, it would be much helpful for later work

Reply:
We have some guide docs for general develop and code review. Please check:
- https://www.notion.so/comcom/ComCom-Developer-s-Guide-ad57ebc31ef94040a4f6e71d9c2042c5
- https://www.notion.so/comcom/ComCom-Code-Review-Guide-514844fe00624e2486c57d5791ed9a2c

For styles, we are referencing some well-known guides available publicly: https://www.notion.so/comcom/ComCom-Code-Review-Guide-514844fe00624e2486c57d5791ed9a2c#870e41d2e23f4e05a420f7cfaf90430e 
But we are trying to have flexibilities. Please feel free to share if you have some ideas in detailed cases.